RAMP/unpacker.py

```diff
@@ -27,18 +27,23 @@ def __str__(self):
 
 
 def unpack(bundle, max_bundle_size_mb=None):
-    # type: (IO[bytes], Optional[int]) -> Tuple[Dict[str, Any], IO[bytes]]
+    # type: (IO[bytes], Optional[int]) -> Tuple[Dict[str, Any], IO[bytes], Dict[str, IO[bytes]]]
     """
     Unpacks a bundled module, performs sanity validation on bundle.
-    both the module metadata and the actual module are returned
+    the module metadata, the actual module and bundle deps are returned
     :rtype: tuple
     """
+    deps_files = dict()
     try:
         with ZipFile(bundle) as zf:
             _validate_zip_file(zf, max_bundle_size_mb)
             metadata = json.load(zf.open('module.json'))
             module = zf.open(metadata["module_file"])
             _validate_metadata(metadata)
+            for filename in zf.namelist():
+                if filename in ['module.json', metadata["module_file"], "deps/"]:
+                    continue
+                deps_files[filename] = zf.open(filename)
```

Comment:
Maybe I misunderstood but do you assume that any file that is not `module.json`, `metadata["module_file"]`, `deps/` is a dependency? Why don't we make it explicit and only take files which inside `/deps`?

Reply:
before getting to this code, I validated that there is exactly 3 parts to the zf 'module.json', metadata["module_file"], "deps/".. 
so there is no chance for any other files.
 
when I have files inside deps/, lets say x and y: 
zf.namelist() will return

module.json
metadata["module_file"]
deps/
deps/x
deps/y

so I actually only adding deps that under deps dir